File: app/src/main/java/com/joshuahalvorson/safeyoutube/database/Playlist.java
package com.joshuahalvorson.safeyoutube.database;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Playlist {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @ColumnInfo(name = "playlist_id")
    public String playlistId;

    @ColumnInfo(name = "playlist_name")
    public String playlistName;

    @ColumnInfo(name = "playlist_video_count")
    public int playlistVideoCount;

    public Playlist(String playlistId, String playlistName, int playlistVideoCount) {
        this.playlistId = playlistId;
        this.playlistName = playlistName;
        this.playlistVideoCount = playlistVideoCount;
    }
}

File: app/src/main/java/com/joshuahalvorson/safeyoutube/Kotlin/database/Playlist.kt
package com.joshuahalvorson.safeyoutube.Kotlin.database

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
data class Playlist(
        @ColumnInfo(name = "playlist_id") var playlistId: String,
        @ColumnInfo(name = "playlist_name") var playlistName: String,
        @ColumnInfo(name = "playlist_video_count") var playlistVideoCount: Int,
        @ColumnInfo(name = "playlist_thumbnail") var playlistThumbnail: String){
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0
}


